woocommerce/woocommerce
# Migrate shipping shipping in product editor to slot fill

### All Submissions:






### Changes proposed in this Pull Request:



Transitioning shipping section in product editor to utilize our slot-fill components. 

Closes #36421  .




### How to test the changes in this Pull Request:

1. Checkout branch.
2. Enable product editor at WooCommerce -> Settings -> Advanced -> Features -> Try the new product editor
3. Go to product editor at Products -> Add new (MVP)
4. Do an overall smoke test of the shipping section, ensure values are persisted and the shipping class modal works as expected.



### Other information:




### FOR PR REVIEWER ONLY:
